packages/flutter_tools/templates/cocoapods/Podfile-swift

```diff
@@ -1,32 +1,41 @@
 # Uncomment this line to define a global platform for your project
 # platform :ios, '9.0'
 
-if ENV['FLUTTER_FRAMEWORK_DIR'] == nil
-  abort('Please set FLUTTER_FRAMEWORK_DIR to the directory containing Flutter.framework')
-end
-
-target 'Runner' do
-  use_frameworks!
-
-  # Pods for Runner
-
-  # Flutter Pods
-  pod 'Flutter', :path => ENV['FLUTTER_FRAMEWORK_DIR']
-
-  if File.exists? '../.flutter-plugins'
-    flutter_root = File.expand_path('..')
-    File.foreach('../.flutter-plugins') { |line|
-      plugin = line.split(pattern='=')
+def parse_KV_file(file,seperator='=')
+  file_abs_path = File.expand_path(file)
+  if !File.exists? file_abs_path
+    puts file_abs_path+"Not exists"
```

Comment:
We should add a message here saying `Generated.xcconfig` or `.flutter-plugins` expected but not present. If you're running `pod install` manually, make sure `flutter build` or `flutter run` is executed once first. 

Then abort instead of letting it potentially finish and have subtle bugs